extension/js/agent/patches/util/addCidToComponent.js

```diff
@@ -8,7 +8,7 @@
   */
 
   Agent.addCidToComponent = function(object) {
-    Agent.setHiddenProperty(object, 'cid',  _.uniqueId('c'));
+    if (!object.cid) Agent.setHiddenProperty(object, 'cid',  _.uniqueId('c'));
```

Comment:
Old Marionette versions does not have cid like Region so it needs a fake cid to be identified in inspector.